cdap-integration-test/src/main/java/co/cask/cdap/test/remote/RemoteWorkflowManager.java

```diff
@@ -60,6 +62,15 @@ public RemoteWorkflowManager(WorkflowId programId, ClientConfig clientConfig, RE
   @Override
   public List<ScheduleSpecification> getSchedules() {
     try {
+      return Schedulers.toScheduleSpecs(getProgramSchedules());
```

Comment:
it doesn't look like anything in here throws a checked exception.
Why do we need the `Throwables.propagate`?